code/game/machinery/rechargestation.dm

```diff
@@ -35,8 +35,7 @@
 	for(var/obj/item/weapon/stock_parts/manipulator/M in component_parts)
 		repairs += M.rating - 1
 	for(var/obj/item/weapon/stock_parts/cell/C in component_parts)
-		recharge_speed *= C.maxcharge / 10000
-
+		recharge_speed *= max((C.maxcharge / 10000), 1)

```

Comment:
This isn't any better for a fix, just restore the high capacity cell to cyborg rechargers.